libclc/clc/lib/amdgcn/workitem/clc_get_global_size.cl

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <clc/workitem/clc_get_global_size.h>
```

Comment:
Why are all of these using `#include <>` instead of `#include ""`? Probably should do a bulk fix of these at some point

Reply:
> Why are all of these using `#include <>` instead of `#include ""`? Probably should do a bulk fix of these at some point

it is following existing style. I'll do the bulk fix.